stackdriver/client.go

```diff
@@ -180,6 +180,21 @@ func (c *Client) Store(req *monitoring.CreateTimeSeriesRequest) error {
 					return
 				}
 				switch status.Code() {
+				// codes.DeadlineExceeded:
+				//   It is safe to recover and retry
+				//   for all cases of
+				//   google.monitoring.v3.MetricService.CreateTimeSeries.
+				//
+				//   On the client side, if the client is unable to
+				//   send the request to server, it's safe to retry
+				//   until the client can send the request to server.
+				//
+				//   On the server side, no sample is unrecoverable
+				//   permanently within the server side timeout.
```

Comment:
I don't think extra explanation on the client & server side is necessary, so I'd drop these two sections.

Reply:
Dropped these two sections. Please take another look.